synapse/lib/stormtypes.py

```diff
@@ -392,6 +392,58 @@ async def _fire(self, name, **info):
         s_common.reqjsonsafe(info)
         await self.runt.snap.fire('storm:fire', type=name, data=info)
 
+class LibPs(Lib):
+
+    def addLibFuncs(self):
+        self.locls.update({
+            'kill': self._kill,
+            'list': self._list,
+        })
+
+    async def _kill(self, prefix):
+        idens = []
+
+        todo = s_common.todo('ps', self.runt.user.iden)
+        tasks = await self.dyncall('cell', todo)
+        for task in tasks:
+            iden = task.get('iden')
+            if iden.startswith(prefix):
+                idens.append(iden)
+
+        if len(idens) == 0:
+            mesg = 'Provided iden does not match any processes.'
+            raise s_exc.StormRuntimeError(mesg=mesg, iden=prefix)
+
+        if len(idens) > 1:
+            mesg = 'Provided iden matches more than one process.'
+            raise s_exc.StormRuntimeError(mesg=mesg, iden=prefix)
+
+        todo = s_common.todo('kill', self.runt.user.iden, idens[0])
+        return await self.dyncall('cell', todo)
+
+    async def _list(self, verbose=False):
+
+        MAXFIELDLEN = 120
+
+        def clamp(field):
```

Comment:
This feels like a display problem, not a list API problem. We should be able to do the clamping in the display logic.